### ISSUE TITLE
TypeError: Gtk.CssProvider.load_from_data() takes exactly 3 arguments (2 given)

### ISSUE THREAD

**Issue by yochananmarqos** [aleiepure/devtoolbox#4]
**Describe the issue**
After building from source using my [PKGBUILD](https://github.com/yochananmarqos/pkgbuilds/blob/master/devtoolbox/PKGBUILD), this is the output running `devtoolbox`:

```
Traceback (most recent call last):
  File "/usr/share/devtoolbox/devtoolbox/main.py", line 73, in do_activate
    win = DevtoolboxWindow(self.debug, application=self)
  File "/usr/share/devtoolbox/devtoolbox/window.py", line 195, in __init__
    "child": ContrastCheckerView(),
  File "/usr/share/devtoolbox/devtoolbox/views/contrast_checker.py", line 28, in __init__
    self._set_example()
  File "/usr/share/devtoolbox/devtoolbox/views/contrast_checker.py", line 47, in _set_example
    lbl_css_provider.load_from_data(lbl_css_str.encode("utf-8"))
TypeError: Gtk.CssProvider.load_from_data() takes exactly 3 arguments (2 given)
```

**Operating system (please complete the following information):**
 - Distro: Manjaro (unstable branch)
 - Version: rolling
 - Desktop Environment: GNOME 44.beta
 - Xorg/Wayland: Xorg

**Comment by tfuxu** [comment]
This is a bug in Pygobject itself. There is an [MR](https://gitlab.gnome.org/GNOME/pygobject/-/merge_requests/231) available with a fix, but it hasn't been yet merged to master. I'd suggest to temporary manually apply a patch in code, like it's done for example in https://github.com/GradienceTeam/Gradience/commit/c5078fa115bb4247f6a9b6f0e998e22f79ebe817.

**Comment by tfuxu** [comment]
This is a bug in Pygobject itself. There is an [MR](https://gitlab.gnome.org/GNOME/pygobject/-/merge_requests/231) available with a fix, but it hasn't been yet merged to master. I'd suggest to temporary manually apply a patch in code, like how it's done for example in https://github.com/GradienceTeam/Gradience/commit/c5078fa115bb4247f6a9b6f0e998e22f79ebe817.

**Comment by yochananmarqos** [comment]
Sorry, forgot to mention I'm testing GNOME 44.beta and installing 44.rc packages as they are released.

I just applied the upstream MR to `python-gobject` 3.43.1 and  it resolved the issue.

There's no issue with `python-gobject` 3.42.2 on my other PC running GNOME 43.3. Sorry, forgot to test before creating the issue.

FYI, `webkitgtk-6.0` is replacing `webkit2gtk-5.0` soon.

**Comment by tfuxu** [comment]
You should leave your issue opened for now, as we don't know if that MR will get merged before GTK 4.10 release, and if it doesn't, the dev thanks to your issue could see that they just need to apply a simple patch in devtoolbox's code to fix it.

**Comment by yochananmarqos** [comment]
@tfuxu That's a good point.

**Comment by aleiepure** [comment]
Fixed in 4da75ee.

**Comment by yochananmarqos** [comment]
Not quite.

```
Traceback (most recent call last):
  File "/usr/bin/devtoolbox", line 48, in <module>
    from devtoolbox import main
  File "/usr/share/devtoolbox/devtoolbox/main.py", line 11, in <module>
    gi.require_version('WebKit2', '5.0')
  File "/usr/lib/python3.11/site-packages/gi/__init__.py", line 129, in require_version
    raise ValueError('Namespace %s not available for version %s' %
ValueError: Namespace WebKit2 not available for version 5.0
```
